src/Workspaces/VisualBasic/Portable/Formatting/DefaultOperationProvider.vb

```diff
@@ -94,27 +94,29 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Formatting
             Return Nothing
         End Function
 
-        Private Function IsSingleLineIfOrElseClauseSyntax(node As SyntaxNode) As Boolean
+        Private Shared Function IsSingleLineIfOrElseClauseSyntax(node As SyntaxNode) As Boolean
             Return TypeOf node Is SingleLineIfStatementSyntax OrElse TypeOf node Is SingleLineElseClauseSyntax
         End Function
 
-        Private Function ColonOrLineContinuationTrivia(trivia As SyntaxTrivia) As Boolean
+        Private Shared Function ColonOrLineContinuationTrivia(trivia As SyntaxTrivia) As Boolean
             Return trivia.RawKind = SyntaxKind.ColonTrivia OrElse trivia.RawKind = SyntaxKind.LineContinuationTrivia
         End Function
 
-        Private Function ColonTriviaFollowedByLineContinuation(triviaList As IEnumerable(Of SyntaxTrivia)) As Boolean
```

Comment:
so.. there seems codestyle project that has duplicated code? should that be changed as well?

Reply:
or. is that linked file?

Reply:
It uses a linked file 👍